host-interaction/process/enumerate-blocking-processes.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: enumerate blocking processes
```

Comment:
this name isn't *super* descriptive to me: its not clear what the process is blocking. would you at least add a description that explains what this is?